test/blackbox-tests/test-cases/pkg/ocaml-compiler.t

```diff
@@ -0,0 +1,110 @@
+Setting the compiler in the lock directory
+
+We need some data for ocamlc -config
+
+  $ mkdir stdlib && touch stdlib/Makefile.config
```

Comment:
Why do we need this file/folder?

Reply:
The `Ocaml_config` module requires it to be there. I don't know if that requirement is justified, but it is present for non OCaml installations.